### DISCUSSION TITLE
In the comments array, I need to replace the keys with the date the comment was created, is it possible to do this?

### DISCUSSION THREAD

**Discussion by zabavnikov** [rebing/graphql-laravel#937] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8766240/186678396-55f08b65-a3f1-450b-8e3b-babc65a493ba.png)

I tried to do this in the CommenQuery, but the array is returned with the usual keys.

```php
return $comments->limit($args['limit'])->get()->mapWithKeys(function ($item, $key) {
    $stack = Carbon::parse($item['created_at'])->format('Ymd');
    return [$stack => $item];
})->all();
```

**Comment by mfn** [comment, marked as answer]
No, that's not possible.

But that's not a limit of neither this library or the underlying GraphQL PHP library: it's a shortcoming in the official GraphQL specification.
